Lib/pathlib.py

```diff
@@ -795,7 +795,11 @@ def name(self):
 
     @property
     def suffix(self):
-        """The final component's last suffix, if any."""
+        """
+        The final component's last suffix, if any.
+
+        This includes the leading dot. For example: '.txt'
```

Comment:
The `.` character is most often called a "period" rather than a "dot", and I suggest using "period" here as well.

"dot" is ambiguous; there are other dot characters, e.g. ["interpunct" a.k.a. "middle dot"](https://en.wikipedia.org/wiki/Interpunct).

While this page in the docs does refer to periods as "dots" in one other place, in the [os.path.splitext() docs](https://docs.python.org/3/library/os.path.html#os.path.splitext) they are indeed called "periods".

Reply:
Done.